app.go

```diff
@@ -14,13 +14,16 @@ type Config struct {
 	HeaderAuth bool
 }
 
+// Deprecated: will be replace with a different object in v2
```

Comment:
we can be more specific
```suggestion
// Deprecated: will be rename to `Passage` in v2
```

Reply:
made this change